tests/testthat/test_may_load.R

```diff
@@ -105,12 +105,18 @@ test_that("Load data have the correct number of significant digits", {
 })
 
 test_that("There are no duplicate values", {
-  result <- validate::check_that(temp_mloads, 
-                                
-                                 length(unique(paste(temp_mloads$SITE_ABB,temp_mloads$CONSTIT,temp_mloads$mod1,temp_mloads$WY,sep="_")))==nrow(temp_mloads)   
-  )
-  
-  expect_no_errors(result)
-  
+
+   mloads_without_ignored_modtypes <- subset(mloads, !(MODTYPE %in% c('COMP', 'CONTIN')))
+   unique_columns <- mloads_without_ignored_modtypes[c('SITE_QW_ID', 'CONSTIT', 'WY')]
+   duplicates <- unique_columns[duplicated(unique_columns),]
+   if(nrow(duplicates) > 0) {
```

Comment:
The intent of this conditional block is to prepare a helpful error message - one that starts with a descriptive error message and ends with R's handy native representation of a data frame.